lightkurve/targetpixelfile.py

```diff
@@ -772,7 +772,14 @@ def update(cadence, log_stretch):
             else:
                 vert.update(line_alpha=0)
                 fig2_dat.data_source.data['image'] = [self.flux[0, :, :] * np.NaN]
-            push_notebook()
+            try:
+                push_notebook()
+            except AttributeError:
+                print('Interact tool must be run in a Jupyter Notebook.')
```

Comment:
Let's use the new nifty logging feature instead of print statements. You should be able to do
```
log.error('Interact tool must be run in a Jupyter Notebook.\n')
```

instead of the print statements.